frontend/src/components/WineDescription.jsx

```diff
@@ -12,26 +13,33 @@ const WineDescription = (props) => {
         `${process.env.REACT_APP_API_ENDPOINT_GET_WINES}/single/${wine_id}`
       );
       setSingleWineData(response.data);
-      // console.log(response.data);
+      console.log(response.data);
     };
     getData();
   }, [wine_id]);
 
+  if (!singleWineData) {
+    return null;
+  }
+
+  const wineObject = singleWineData.wines[0];
+  console.log(wineObject);
```

Comment:
I recommend start removing the console logs and even commented out console logs from code is going to master.